templates/order/form-tracking.php

```diff
@@ -20,7 +20,7 @@
 global $post;
 ?>
 
-<form action="<?php echo esc_url( get_permalink( $post->ID ) ); ?>" method="post" class="track_order">
+<form action="<?php echo esc_url( get_permalink( $post->ID ) ); ?>" method="post" class="woocommerce-form woocommerce-form-track-order">
```

Comment:
This seems like a good idea for consistency. For backwards compatibility, the old `track_order` class should still be kept and these new classes should be added in addition to that class, otherwise themes that currently style the `track_order` class will have broken styling when updated.

Please add the old class, update the version tag at the top of the file to 3.5.0, and this will be good to merge. Thanks.